src/com/mambu/apisdk/MambuAPIService.java

```diff
@@ -101,10 +106,33 @@ public String executeRequest(String urlString, ParamsMap params, Method method)
 	 */
 	public String executeRequest(String urlString, ParamsMap params, Method method,
 			RequestExecutor.ContentType contentTypeFormat) throws MambuApiException {
+
 		return executor.executeRequest(urlString, params, method, contentTypeFormat);
 	}
 
 	/**
+	 * Delegates the execution to a RequestExecutor
+	 * 
```

Comment:
May be we can add here a bit more details about the purpose of this particular executeRequest